src/main/java/com/ubb/jobs/repo/impl/JobRepo.java

```diff
@@ -51,4 +51,20 @@ public JobDto getOne(Integer id) {
     public JobDto findJobById(Integer id) {
         return jobMapper.toDto(jpaJobRepo.findJobById(id));
     }
+
+    public JobDto editJob(Integer id, JobDto newJob){
+        Job job = jobRepo.getOne(id);
+        job.setTitle(newJob.getTitle());
+        job.setPeriodStart(newJob.getPeriodStart());
+        job.setPeriodEnd(newJob.getPeriodEnd());
+        job.setDescription(newJob.getDescription());
+        job.setPeopleRequired(newJob.getPeopleRequired());
+        job.setStatus(newJob.getStatus());
+        job.setStartTime(newJob.getStartTime());
+        job.setEndTime(newJob.getEndTime());
+        job.setHoursPerDay(newJob.getHoursPerDay());
+        job.setHoursPerWeek(newJob.getHoursPerWeek());
+        job.setJobAbilities(newJob.getJobAbilities());
+        return jobMapper.toDto(job);
+    }
```

Comment:
nu o sa functioneze, o sa fac eu partea asta pentru ca ii posibil sa fie complicatii. Ce ai facut e doar sa editezi valoarea locala a obiectului Job, fara sa faci modificari in db.

Reply:
ok, ma gandeam ca nu va fi asa simplu..